packages/react/src/autocomplete/Autocomplete.test.tsx

```diff
@@ -1,28 +1,28 @@
 import '@testing-library/jest-dom';
 import 'html-validate/jest';
+import { useRef } from 'react';
 import userEvent from '@testing-library/user-event';
-import { render, cleanup, act, waitFor } from '../../../../test-utils';
-import { Autocomplete } from './Autocomplete';
+import {
+	render,
+	cleanup,
+	act,
+	waitFor,
+	renderHook,
+} from '../../../../test-utils';
+import { STATE_OPTIONS, Option } from '../combobox/test-utils';
+import { Autocomplete, AutocompleteProps } from './Autocomplete';
 
 afterEach(cleanup);
 
-const mockLoadOptions = jest.fn().mockResolvedValue([
```

Comment:
This list of states has been duplicated across the code base, so it's been consolidated as part of this PR